### ISSUE TITLE
batch-norm project: use softmax_cross_entropy_with_logits for MNIST classification 

### ISSUE THREAD

**Issue by lwohlhart** [udacity/deep-learning#218]
In the notebooks demonstrating the batch normalization we are classifying MNIST digits.
Since this is the reference implementation and the classification output should be a probability distribution over the available classes we should use **softmax_cross_entropy_with_logits** instead of **sigmoid_cross_entropy_with_logits** for the model_loss in the train()-functions
```
def train(..):
    ...
    # Define loss and training operations
    model_loss = tf.reduce_mean(tf.nn.sigmoid_cross_entropy_with_logits(logits=logits, labels=labels))
    train_opt = tf.train.AdamOptimizer(learning_rate).minimize(model_loss)
    ...
```
```
def train(..):
    ...
    # Define loss and training operations
    model_loss = tf.reduce_mean(tf.nn.softmax_cross_entropy_with_logits(logits=logits, labels=labels))
    train_opt = tf.train.AdamOptimizer(learning_rate).minimize(model_loss)
    ...
```

This concerns

https://github.com/udacity/deep-learning/blob/master/batch-norm/Batch_Normalization_Exercises.ipynb
https://github.com/udacity/deep-learning/blob/master/batch-norm/Batch_Normalization_Solutions.ipynb

**Comment by abhiojha8** [comment]
Though we are a little late to respond, we encourage such _user-specific_ issues to be raised on the Knowledge hub, so that our experienced mentors can chime-in and new students can also learn from the discussion thread. 
Thank you
